docs/system/k8s-metrics.md

```diff
@@ -0,0 +1,168 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: Container
+--->
+
+# Semantic Conventions for K8s Metrics
```

Comment:
We are not consistent in other docs using K8s vs Kubernetes but I think on the title it makes sense to use Kubernetes as we do here https://github.com/open-telemetry/semantic-conventions/blob/main/docs/resource/k8s.md